File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/user/exception/UserNotFoundException.java
package band.gosrock.domain.domains.user.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class UserNotFoundException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new UserNotFoundException();

    private UserNotFoundException() {
        super(UserErrorCode.USER_NOT_FOUND);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/user/exception/UserNotFoundException.kt
package band.gosrock.domain.domains.user.exception

import band.gosrock.common.exception.DuDoongCodeException

class UserNotFoundException private constructor() : DuDoongCodeException(UserErrorCode.USER_NOT_FOUND) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = UserNotFoundException()
    }
}
